src/parser/parser.yy

```diff
@@ -1048,7 +1048,17 @@ opt_argument_list
     ;
 
 argument_list
-    : expression {
+    : KW_VERTEX {
```

Comment:
Do we have some exception tests for this syntax? like:
```
Go from id(vertex) over *;

Match (n:player) return id(vertex);
```
We should report error in these scenarios.

Reply:
done